app/jobs/fees/create_pay_in_advance_job.rb

```diff
@@ -2,8 +2,12 @@
 
 module Fees
   class CreatePayInAdvanceJob < ApplicationJob
+    include ConcurrencyThrottlable
```

Comment:
Do we need to restrict it to only 5 concurrent calls? If not we can remove this line.
Anrok doesn't have concurrency restrictions, only 10 requests per second per API key.

Reply:
removed